src/core/public/public.api.md

```diff
@@ -119,7 +121,23 @@ export interface ChromeDocTitle {
 }
 
 // @public (undocumented)
-export type ChromeHelpExtension = (element: HTMLDivElement) => () => void;
+export interface ChromeHelpExtension {
+    // (undocumented)

```

Comment:
A lot of "Warning"s and "(undocumented)" bits in here and I'm not super certain what each means but it certainly seems scary. Can any of them be resolved?

Reply:
Hmmm, let me see what happens if I export all those types